### PR TITLE
Feature/Moderation tools

### PR DESCRIPTION
**Pull request by avcio9**
- [ ] Save on mongo
- [x] Remove discord hook

### CODE REVIEW

**Comment by avcio9** [comment]
No need to store the spectator vehicle, with the spectator base you have to disconnect from server in order to get out of it

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "debug": "4.3.4",
+    "discord-webhook-node": "^1.1.8",
```
Comment (QuentinGruber) [inline package.json:18]: should be a fixed version since it's a third-party dependency "discord-webhook-node": "1.1.8" and rerun `npm i` to updt lockfile.

**Review by QuentinGruber — CHANGES_REQUESTED** [review]

**Review by QuentinGruber — CHANGES_REQUESTED** [review]

File: src/servers/ZoneServer2016/zoneserver.ts [hunk 2]
```diff
@@ -2922,7 +2983,7 @@ export class ZoneServer2016 extends EventEmitter {
   vehicleManager(client: Client) {
     for (const key in this._vehicles) {
       const vehicle = this._vehicles[key];
-      if (vehicle.vehicleId == VehicleIds.SPECTATE) continue; //ignore spectator cam
+      //if (vehicle.vehicleId == VehicleIds.SPECTATE) continue; //ignore spectator cam // shouldnt be needed anymore
```
[the remark this answers is not in the document]
Comment (avcio9) [reply src/servers/ZoneServer2016/zoneserver.ts:2986]: not, spectator vehicle isnt stored anymore

**Comment by QuentinGruber** [comment]
If you hit a dead body with a shotgun it trigger fairplay detection

**Comment by Z1Meme** [comment]
@avcio9 the merge conflicts on this one are horrendous, just copy-paste dev zonepackethandlers and re-add your work i guess, idk what all you changed so I can't really fix the conflicts myself, ignore my last commit because it didn't do anything

**Comment by Z1Meme** [comment]
I'll fix the errors some time this week since a lot of them are caused by the zonepackethandlers change

**Comment by Z1Meme** [comment]
yep it's fucked

**Comment by Z1Meme** [comment]
@QuentinGruber can't login to the server due to unknown flag not setting off ClientIsReady idk why

**Comment by Z1Meme** [comment]
image
sideways gate / expansion bug still works (need to validate placement rotatio, should just ignore the rotation the client gives for some static rotation objects and just use a set value)

**Comment by Z1Meme** [comment]
@QuentinGruber This can be merged now, it's tested with no major issues, the above issues can be fixed later.

**Review by QuentinGruber — APPROVED** [review]

**Pull request merged**